deploy/kubelet-config/03-kubelet-config.yaml

```diff
@@ -10,3 +10,5 @@ spec:
     systemReserved:
       cpu: 1000m
       memory: 1Gi
+    imageMinimumGCAge: 30 # https://issues.redhat.com/browse/OSD-3010
+    imageGCHighThresholdPercent: 35 # https://issues.redhat.com/browse/OSD-3010
```

Comment:
And this is going from 85% down to 35? This seems like a significant drop, especially without changing `ImageGCLowThresholdPercent`